backward.hpp

```diff
@@ -24,6 +24,8 @@
 #ifndef H_6B9572DA_A64B_49E6_B234_051480991C89
 #define H_6B9572DA_A64B_49E6_B234_051480991C89
 
+#include "backward_config.hpp"
```

Comment:
now backward requires a file named "backward_config.hpp". My goal was to keep this library as standalone and easy to include as possible. I am not against making the CMakeFile more clever, but it should still be possible to `cp backward.{hpp,cpp} my_src/` without being dependent of CMake.

Is going trough a config.hpp header really better than adding `#define`s to the compiler command line as it is done right now?